CHANGELOG.md

```diff
@@ -12,7 +12,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Maintenance
 ### Refactoring
 
-## [Unreleased 2.x](https://github.com/opensearch-project/geospatial/compare/2.9...2.x)
+## [Unreleased 2.x](https://github.com/opensearch-project/geospatial/compare/2.10...2.x)
 ### Features
```

Comment:
You need to delete all logs from here when you move them to release note.

Reply:
Since the code freeze is not done, I am keeping these changes.

Reply:
Plus 2.11 is not ready. I don't think so we should make this change

Reply:
deleted.